src/environment_setup/python_package.ml

```diff
@@ -72,5 +72,5 @@ let default ~host ~run_program ~install_path () =
     create_python_tool ~host ~run_program ~install_path
       ~version:"0.0.21" (Pip, Package_PyPI "topiary");
     create_python_tool ~host ~run_program ~install_path
-      ~version:"0.3.1" (Pip, Package_PyPI "vaxrank");
+      ~version:"0.4.0" (Pip, Package_PyPI "vaxrank");
```

Comment:
I thought we were just going to turn this on for FFPE samples to allow for the remote possibility of detecting alternate splicing after the variant. Since we haven't yet actually seen this it must be at least moderately rare for a variant to cause downstream splicing which still aligns well enough for us to detect.